tests/contrib/asyncio/test_tracer.py

```diff
@@ -85,3 +85,69 @@ def test_context_task_none(self):
         # but a new Context is still created making the operation safe
         ctx = self.tracer.get_call_context()
         ok_(ctx is not None)
+
+    @mark_asyncio
+    def test_exception(self):
+        async def f1():
+            with self.tracer.trace('f1'):
+                raise Exception('f1 error')
+
+        try:
+            with self.assertRaises(Exception):
+                yield from f1()
+        finally:
+            traces = self.tracer.writer.pop_traces()
+            eq_(1, len(traces))
+            spans = traces[0]
+            eq_(1, len(spans))
+            span = spans[0]
+            eq_(1, span.error)
+
+    @mark_asyncio
+    def test_nested_exceptions(self):
+        async def f1():
+            with self.tracer.trace('f1'):
+                raise Exception('f1 error')
```

Comment:
can you also add a test for the ``aiohttp``? it's enough that a test raises an exception like ``1/0`` without handling. You can check it there: https://github.com/DataDog/dd-trace-py/blob/palazzem/async-python/tests/contrib/aiohttp/test_middleware.py#L40

Also can you check the error message and the stack trace?
```python
span.get_tag('error.msg')
# 'f1 error'
span.get_tag('error.stack')
# 'Traceback (most recent call last):\n  File ".../tests/contrib/asyncio/test_tracer.py", line 93, in f1\n    raise Exception(\'f1 error\')\nException: f1 error\n'
```
for the stack you can use something like:
```python
ok_('Exception: f1 error' in span.get_tag('error.stack'))
```